src/main/webapp/assets/js/feedScript.js

```diff
@@ -785,21 +838,44 @@ function resetMarks(formElements) {
 function markInvalidInputs(invalidIds, errorMessages, formElements) {
   resetMarks(formElements);
   invalidIds.forEach((id) => {
-    const elt = formElements.namedItem(id);
+    const elt = document.getElementById(id);
     if (elt) {
-      elt.style.background = '#ff999966';
+      elt.style.backgroundColor = '#ff999966';
     }
   });
   if (errorMessages.length > 0) {
     const modalError = document.createElement('div');
     modalError.setAttribute('id', 'modal-input-error');
     modalError.setAttribute('tabindex', '0');
-    modalError.innerHTML += '<ul>';
+    let errorHTMLString = '<ul>';
     errorMessages.forEach((message) => {
-      modalError.innerHTML += '<li>' + message + '</li>';
+      errorHTMLString += '<li>' + message + '</li>';
     });
-    modalError.innerHTML += '</ul>';
-    modalForm.insertBefore(modalError, submitModalButton);
+    errorHTMLString += '</ul>';
+    modalError.innerHTML = errorHTMLString;
+    modalButtonDiv.insertBefore(modalError, submitModalButton);
+  }
+}
+
+/**
+ * Adds an error message if the file is larger than 4MB or is not an image.
+ * @param {array} invalidIds
+ * @param {array} errorMessages
+ * @return {void}
```

Comment:
(Copied from another PR)
for this, you could do

```js
/*
...
* @param {Array<string>} invalidIds - some description
* @param {Array<string>} errorMessages - some description
*/
```
to be more specific! Also, having @return {void} is optional, since void doesn't really exist as a type in Javascript anyway (it's an operator that always returns undefined after evaluating a statement https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void)

Reply:
Great, thank you!